tools/cli/test/data/changelog/new-api-version/base/openapi-2023-01-01.json

```diff
@@ -946,7 +946,7 @@
         } ],
         "summary" : "Return One Identity Provider by ID",
         "tags" : [ "Federated Authentication" ],
-        "x-sunset" : "2025-01-01"
```

Comment:
should we use the same dates from https://github.com/10gen/mms/pull/110515 ?

Reply:
maybe we can also add a ticket to add a filter functionality per checker to changelog, so that we could exclude/ignore the removed entries similar to what we use for the exemptions

Reply:
No, we shouldn’t do that. These files are used for e2e tests, and if we set that date, the tests will start failing again in 2026.